torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -586,15 +592,7 @@ def _pad_with_vector_fill(
 def pad_segmentation_mask(
     segmentation_mask: torch.Tensor, padding: Union[int, List[int]], padding_mode: str = "constant"
 ) -> torch.Tensor:
-    num_masks, height, width = segmentation_mask.shape[-3:]
```

Comment:
Not sure why we replicated `pad_image_tensor` here if the only difference is that we set a fixed `fill=0`.